kubectl-rook-ceph.sh

```diff
@@ -445,13 +651,20 @@ function run_start_debug() {
   echo "setting debug command to main container"
   deployment_spec=$(update_deployment_spec_command "$deployment_spec")
 
+  # scale down the daemon pod if it's running
+  set +e
+  echo "get pod for deployment $deployment_name"
   deployment_pod=$(KUBECTL_NS_CLUSTER get pod | grep "$deployment_name" | awk '{ print $1  }')
-  # scale the deployment to 0
-  KUBECTL_NS_CLUSTER scale deployments "$deployment_name" --replicas=0
-
-  # wait for the deployment pod to be deleted
-  echo "waiting for the deployment pod \"$deployment_pod\" to be deleted"
-  KUBECTL_NS_CLUSTER wait --for=delete pod/"$deployment_pod" --timeout=60s
+  set -e
+  if [ "$deployment_pod" != "" ]; then
+    # scale the deployment to 0
```

Comment:
scaling deployment to 0 can be still be out of `if` statement

Reply:
If there is no pod, is there any reason to still scale it to 0? Are you just saying it's a precaution to still scale it down?

Reply:
Yes for a precaution,
A pod can be not present for certain interval because of some restarting, etc
But the deployment may exist